velox/connectors/hive/TableHandle.h

```diff
@@ -45,12 +47,14 @@ class HiveColumnHandle : public ColumnHandle {
       ColumnType columnType,
       TypePtr dataType,
       TypePtr hiveType,
-      std::vector<common::Subfield> requiredSubfields = {})
+      std::vector<common::Subfield> requiredSubfields = {},
+      ValueType valueType = ValueType::kDefault)
```

Comment:
I think we should follow the HiveTableHandle and add a generic argument like
 `const std::unordered_map<std::string, std::string>& columnParameters = {}`

We can then define `struct ColumnParameter` similar to `struct TableParameter` and add an entry for our need.

Reply:
@Yuhta Do you agree with this API approach?